migrator/zenodo_rdm_migrator/transform/entries/users.py

```diff
@@ -47,6 +47,16 @@ def _confirmed_at(self, entry):
         """Returns the confirmation date."""
         return entry.get("confirmed_at")
 
+    def _blocked_at(self, entry):
+        """Returns the blocking/inactivation date."""
+        if not entry["active"]:
+            return datetime.utcnow().isoformat()
```

Comment:
This might make us loose some insight on when the blocking happened. What about using the last login information? (and if not available, default to now)

Reply:
Agree, didn't think of it at the time :)

Yeah, I can use this date or worst case even the email confirmation date.